docs/security-council-manager.md

```diff
@@ -0,0 +1,59 @@
+# Security Council Manager as a source of truth
+
+The Security Council Manager is the source of truth for the membership of all Security Councils on all chains where they are deployed. It contains a registered list of Security Councils, which it will update with any changes to membership.
+
+All changes to membership should be made through the manager, which will then propagate these changes to the councils. Any changes made directly to the councils will be overwritten the next time the Manager pushes an update. As a reminder the normal flow for an election is:
+- **SecurityCouncilNomineeElectionGovernor.createElection** - called at T+0 where T is a multiple of 6 months after the first election date
+- **SecurityCouncilNomineeElectionGovernor.execute** - called at T+21 days
+- **SecurityCouncilMemberElectionGovernor.execute** - called at T + 42 days
+- **ArbitrumTimelock.execute** (on L2) - called at T + 42 days
+- **Outbox.executeTransaction** - called at T + 49 days
+- **L1ArbitrumTimelock.executeBatch** - called at T + 52 days
+- Retryable ticket execution on ArbOne and Nova - also called at T + 52 days
+
+
+## Public functions
+
+### Replace cohort
+
+Can only be called by the Member Election Governor. It is used to replace a whole cohort of (6) members. This is the function that will be called for the standard elections that take place every 6 months.
+
+### Remove member
+
+Can be called by the Emergency Security Council and the Removal Governor. Is used to remove a member. The Constitution allows members to be removed if 9 of 12 members wish them to be, or if the DAO votes to remove a member and 5/6 of voted tokens are in favour of removal.
+
+### Add member
+
+Can only be called by the Emergency Security Council. Can only be called if the there are less than 12 members in the Security Council because one has been removed.
```

Comment:
```suggestion
Can only be called by the Emergency Security Council. Can only be called if the there are less than 12 members in the Security Council because at least one has been removed.
```